crates/edr_provider/src/data.rs

```diff
@@ -3743,5 +3743,10 @@ mod tests {
             chain_id: 1,
             url: get_alchemy_url(),
         },
+        mainnet_cancun2 => {
```

Comment:
Change request: could you add a comment to the code explaining how this test is different from the `mainnet_cancun` test?

Reply:
Addressed in https://github.com/NomicFoundation/hardhat/pull/5100/commits/03796a3f34646b5fb42db7e342149f9c82fe76e5